src/sentry/static/sentry/app/components/group/inboxBadges/eventAnnotation.tsx

```diff
@@ -9,13 +9,22 @@ import Tag from 'app/components/tag';
  */
 
 type Props = {
-  annotation: string;
+  annotation?: string;
```

Comment:
is this prop still used then?

Reply:
it is, either annotation or html annotation is passed.
Annotation for logger, html annotation for group annotations